### DISCUSSION TITLE
Expanded record view [Jun 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2597] [Discord Threads]
> 🙋‍♂️ **JB_Denton**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### for existing record, hover over record ID & click on 'expand row' icon  
![image0](https://cdn.discordapp.com/attachments/991224553147736155/991225271371976714/Screenshot_2022-06-28_at_11.44.24_AM.png)  
#  
> 👨‍🦲 **dstala**  
#### for new record, use `add new row` option on top right menu bar  
![image0](https://cdn.discordapp.com/attachments/991224553147736155/991225495867883580/Screenshot_2022-06-28_at_11.45.35_AM.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991224553147736155)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
